_includes/specials/taproot/en/11-vaults-with-taproot.md

```diff
@@ -29,9 +29,9 @@ due to the use of merkle branches and shorter BIP340 signatures (especially for
 For instance, the *unvault* output script in a multi-party setup with 6 "cold" keys and 3 "active" keys
 (with a threshold of 2) could be represented as a Taproot of depth 2 with leaves:
 
-- `<X> CSV <active key 1> CHECKSIGADD <active key 2> CHECKSIGADD 2 EQUAL`
-- `<X> CSV <active key 2> CHECKSIGADD <active key 3> CHECKSIGADD 2 EQUAL`
-- `<X> CSV <active key 3> CHECKSIGADD <active key 1> CHECKSIGADD 2 EQUAL`
+- `<X> CSV DROP <active key 1> CHECKSIGADD <active key 2> CHECKSIGADD 2 EQUAL`
+- `<X> CSV DROP <active key 2> CHECKSIGADD <active key 3> CHECKSIGADD 2 EQUAL`
+- `<X> CSV DROP <active key 3> CHECKSIGADD <active key 1> CHECKSIGADD 2 EQUAL`
```

Comment:
I think we need to update the cost as well. Also as John pointed out the first `CHECKSIGADD` here should be `CHECKSIG`s, sorry for the overlook i should have double checked the script i sent you.

Reply:
I've updated the scripts, and removed the transaction weight values.